content/threads_intro.tex

```diff
@@ -0,0 +1,73 @@
+This section specifies the interaction between \openshmem{} interfaces and the
+user threads, and also describes the routines that can be used for initializing and 
+querying the thread environment.
+ 
+
+\begin{itemize}
+
+\item
+In a thread-compliant implementation, an \openshmem{} \ac{PE} is multithreaded. 
+Each thread can issue \openshmem{} calls; however the threads
+are not separately addressable. The symmetric heap and symmetric variables scope
+are not impacted by multiple threads invoking the
+\openshmem{} interfaces, i.e., 
+each \ac{PE} has a single symmetric data segment and symmetric heap that is shared by
+all threads within that \ac{PE}.  For example, a thread invoking a memory allocation
+routine such as \FUNC{SHMEM\_MALLOC()} 
+allocates memory that is accesible by all threads of the \ac{PE}. 
+The requirement that the same symmetric heap operations must
+be executed by all processes in the same order also applies in a threaded
+environment. 
+                                    	
+\item In a thread-compliant implementation, 
```

Comment:
This statement's implication seems more direct: A thread-compliant implementation is only one that supports SHMEM_THREAD_MULTIPLE. Is that the intent here?